src/html/mod.rs

```diff
@@ -9,3 +12,63 @@ pub use self::local_name::{LocalName, LocalNameHash};
 pub use self::namespace::Namespace;
 pub use self::tag::Tag;
 pub use self::text_type::TextType;
+
+/// Convert text to HTML
+#[inline]
+pub(crate) fn escape_body_text(mut content: &str, output_handler: &mut impl FnMut(&str)) {
+    loop {
+        if let Some(pos) = memchr3(b'&', b'<', b'>', content.as_bytes()) {
```

Comment:
Nit: let's use `match` here instead of `if`

Reply:
I think this is clearer, because it checks for presence (a binary choice), and not among multiple possibilities. Using `match` for an `if/else` decision seems odd.